nautobot/extras/factory.py

```diff
@@ -202,6 +203,25 @@ def date_done(self, created, extracted, **kwargs):  # pylint: disable=method-hid
                 self.date_done = self.date_created + timedelta(minutes=faker.Faker().random_int())
 
 
+class ScheduledJobFactory(BaseModelFactory):
+    """ScheduledJob model factory."""
+
+    class Meta:
+        model = ScheduledJob
+
+    class Params:
+        has_user = NautobotBoolIterator(chance_of_getting_true=80)
+        has_task_args = NautobotBoolIterator(chance_of_getting_true=10)
+        has_task_kwargs = NautobotBoolIterator(chance_of_getting_true=90)
+
+    name = factory.Sequence(lambda n: f"Scheduled Job {n}")
+    interval = factory.Faker('random_element', elements=['hourly', 'daily', 'weekly'])
+    start_time = factory.Faker('future_datetime', end_date='+30d')
+    enabled = factory.Faker('boolean', chance_of_getting_true=90)
+    user = factory.Maybe("has_user", random_instance(get_user_model()), None)
+    task = factory.Faker("word")
```

Comment:
should we have this choose from actual Job tasks instead of random words?

Reply:
Updated